### DISCUSSION TITLE
id and relay server have already been set.

### DISCUSSION THREAD

**Discussion by mahdinuri** [rustdesk/rustdesk#6046] [General]
Hi.
In Windows, I want to install rustdesk in such a way that ID and relay server are already set on rustdesk.
It means that the user only installs the rustdesk and does not make any settings. But ID and relay server should be set on rustdesk.
How can I do this?
Many thanks

**Comment by MrChenhtlss** [comment]
Totally possible, but you'll need to build your own client

**Reply by pdifeo** [reply]
It's possible with PRO version or following this information
https://rustdesk.com/docs/en/self-host/client-configuration/#automatic-config

**Reply by MrChenhtlss** [reply]
Thank you very much for the solution, this is the best way, but it is possible to modify the code and build your own client but doing this I don't know if it will affect the sales of PRO, so I will not provide the method

**Reply by erland-20** [reply]
I have tried various ways to try to build rustdesk with the ID and relay that I have fixed in playground.yml, but when building why do I always get a lot of errors, I have followed the instructions from the official rustdesk website but it always fails to build. Can someone make a tutorial or try to help me to build?
